src/pages/Browse/index.jsx

```diff
@@ -325,19 +334,36 @@ const Browse = () => {
                       <td>{data?.uploaddate.split(".")[0]}</td>
                     </tr>
                   ))}
-                <tr>
+                <tr className="text-left">
+                  <td colSpan="4" />
                   <td colSpan="6">
```

Comment:
I will still suggest to you complete row as one `td` element rather than breaking it into 2. It will look much cleaner and supports longer pagination.

Reply:
You can see here how long pagination element distorts the view.

https://user-images.githubusercontent.com/18077542/177569635-4b1caf23-b6d3-475a-b463-ac7fd58f89fd.mp4